x-pack/solutions/security/plugins/security_solution/public/detection_engine/rule_creation_ui/components/ai_assistant/index.test.tsx

```diff
@@ -18,22 +18,22 @@ jest.mock('../../../../assistant/use_assistant_availability', () => ({
 }));
 
 const useAssistantAvailabilityMock = useAssistantAvailability as jest.Mock;
-
+const getFields = jest.fn().mockReturnValue({ queryBar: { value: { query: 'something' } } });
```

Comment:
Nit: this could us a "mock" in its name for clarity:

```suggestion
const getFieldsMock = jest.fn().mockReturnValue({ queryBar: { value: { query: 'something' } } });
```